coral-hive/src/main/java/com/linkedin/coral/transformers/CoralRelToSqlNodeConverter.java

```diff
@@ -92,8 +92,24 @@ public boolean requireCastOnString() {
       }
 
       /**
-       * Override this method so that there will be explicit and correct table alias for selected fields, which is necessary for
-       * data type derivation on SqlNodes
+       * Override this method so that there will be explicit table alias for selected fields, which is necessary for
+       * data type derivation on SqlNodes.
+       *
+       * For the following input SQL:
+       * CREATE TABLE default.complex(s struct(name:string, age:int))
+       * SELECT tbl.str.a FROM db.tbl
+       *
+       * With this override, the converted SqlNode is:
+       * SELECT `complex`.`s`.`name` AS `name`
+       * FROM `default`.`complex` AS `complex`
+       *
+       * Without this override, the converted SqlNode is:
+       * SELECT `s`.`name` AS `name`
```

Comment:
Why does the alias show up here in the case of "without the override"?

Reply:
This field alias is not related to table alias generated by this override, this override won't affect the field alias like `name`